CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contribution guidelines
+
+## Filing issues
+
+File issues using the standard [Github issue tracker](https://github.com/redhat-developer/openshift-vsts/issues) for this repository.
```

Comment:
This - https://github.com/redhat-developer/openshift-vsts/issues

Reply:
Bugger, let me fix this ;-)

Reply:
I pushed an update.